pages/linux/lslogins.md

```diff
@@ -0,0 +1,28 @@
+# lslogins
+
+> Show information about users in the Linux system.
+> More information: <https://man7.org/linux/man-pages/man1/lslogins.1.html>.
+
+- Display users in the system:
+
+`lslogins`
+
+- Diplay users in the system that are in groups:
```

Comment:
I'm not sure about this translation maybe because I don't understand the command. :man_shrugging: 
If I run:
```bash
$ lslogins --groups=wheel
```
I get the users belonging to the `wheel` group. So, I think this is better:
```suggestion
- Display users belonging to a specific group:
```
But, your description sounds like: If I run this command I'll get all the users that belong to any group.